### DISCUSSION TITLE
react-native-paper-v5

### DISCUSSION THREAD

**Discussion by EdgarsLv** [callstack/react-native-paper#3242] [Q&A]
Can someone explain what is with v5? 
https://callstack.github.io/react-native-paper/introducing-v5-with-material-you.html

Here it says that latest is v4.12.2. Installing @latest  installs v4.12.2.
So where and how i can get mentioned v5?

**Comment by ArnabXD** [comment]
Current (4.\*) is based on material design 2 and version 5.* is based on material design 3. You can dig into their differences.

You can use v5 using: 

```
yarn add react-native-paper@rc
```

**Reply by gustavohbeg** [reply]
when will v5 officially be released ?

**Reply by cesar-presence** [reply]
> when will v5 officially be released ?

Does any contributor has any idea about that?

**Reply by lukewalczak** [reply]
We've added an `Installation` [section](https://callstack.github.io/react-native-paper/introducing-v5-with-material-you.html#installation) in the mentioned migration guide, describing how to install `v5`.

> when will v5 officially be released?

We are still working on that and it's hard to indicate the specific date.

**Reply by brunohkbx** [reply]
Will both versions continue to be supported or v4 will stop once v5 has been released?

**Reply by lukewalczak** [reply]
> Will both versions continue to be supported or v4 will stop once v5 has been released?

Basically, the stable v5 will replace the v4, however, we are supporting two design system generations as described in the [migration guide](https://callstack.github.io/react-native-paper/introducing-v5-with-material-you.html#versioning), which means that you can use v5 with the previous design system. 
You can think about v5 as an extended and modified _(mostly in terms of theming and typography related to Material You)_ v4.
